### PR TITLE
DOC-4500

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-4500

### CODE REVIEW

**Comment by jamesnocentini** [comment]
assigning @bbrks as reviewer.

File: modules/ROOT/assets/attachments/sync-gateway-admin.yaml [hunk 1]
```diff
@@ -386,13 +386,21 @@ parameters:
...
     description: |
       Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
       panics logging and level=3 will log panics only.
     type: integer
+    deprecated: true
+  logLevel:
+    in: query
+    name: logLevel
+    description: |
+      Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
```
Comment (bbrks) [inline modules/ROOT/assets/attachments/sync-gateway-admin.yaml]: The parameter was changed to accept the same textual values as the SG config. Valid options are `none`, `error`, `warn`, `info`, `debug`, `trace`:

https://github.com/couchbase/docs-sync-gateway/blob/8b15b8fbd002d17636d9099eed06780be847c6f0/modules/ROOT/pages/logging.adoc#log-levels

**Review by bbrks — CHANGES_REQUESTED** [review]
New parameter takes textual log level, e.g. "info", "warn", etc.

File: modules/ROOT/assets/attachments/sync-gateway-admin.yaml (hunk 1 above)
Comment (bbrks) [inline modules/ROOT/assets/attachments/sync-gateway-admin.yaml]:
```suggestion
      Verbosity of the logging. Available values are `none`, `error`, `warn`, `info`, `debug,` or `trace`.
```

File: modules/ROOT/assets/attachments/sync-gateway-admin.yaml [hunk 2]
```diff
@@ -386,13 +386,21 @@ parameters:
...
+      panics logging and level=3 will log panics only.
+    type: integer
```
Comment (bbrks) [inline modules/ROOT/assets/attachments/sync-gateway-admin.yaml]:
```suggestion
    type: string
```

File: modules/ROOT/assets/attachments/sync-gateway-admin.yaml [hunk 3]
```diff
@@ -386,13 +386,21 @@ parameters:
...
       Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
       panics logging and level=3 will log panics only.
     type: integer
+    deprecated: true
+  logLevel:
+    in: query
+    name: logLevel
+    description: |
+      Verbosity of the logging. The default value is 1 which is regular logging. level=2 enables warnings and
+      panics logging and level=3 will log panics only.
```
Comment (bbrks) [inline modules/ROOT/assets/attachments/sync-gateway-admin.yaml]:
```suggestion
```

**Review by bbrks — CHANGES_REQUESTED** [review]
Updating with suggestions

**Comment by jamesnocentini** [comment]
@bbrks Thanks!

**Pull request merged**